testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/SinglenodeSparkIceberg.java

```diff
@@ -57,11 +57,11 @@ public SinglenodeSparkIceberg(Standard standard, Hadoop hadoop, DockerFiles dock
     @Override
     public void extendEnvironment(Environment.Builder builder)
     {
-        builder.configureContainer(HADOOP, dockerContainer -> {
-            dockerContainer.setDockerImageName("ghcr.io/trinodb/testing/hdp3.1-hive:" + hadoopImagesVersion);
-            dockerContainer.withCopyFileToContainer(
-                    forHostPath(dockerFiles.getDockerFilesHostPath("conf/environment/singlenode-hdp3/apply-hdp3-config.sh")),
-                    CONTAINER_HADOOP_INIT_D + "apply-hdp3-config.sh");
+        builder.configureContainer(HADOOP, container -> {
+            container.setDockerImageName("ghcr.io/trinodb/testing/hdp3.1-hive:" + hadoopImagesVersion);
+            container.withCopyFileToContainer(
+                    forHostPath(dockerFiles.getDockerFilesHostPath("conf/environment/singlenode-spark-iceberg/apply-hive-config-for-iceberg.sh")),
+                    CONTAINER_HADOOP_INIT_D + "/apply-hive-config-for-iceberg.sh");
```

Comment:
optional nit: redundant `/`

Reply:
I am aware. I do think that CONTAINER_HADOOP_INIT_D should **not** include trailing slash, and i hope we clean this up some day. The code here is already prepared for that.
cc @wendigo 

(same for CONTAINER_HEALTH_D and CONTAINER_CONF_ROOT)